cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -91,10 +85,6 @@ func (p *BicepProvider) RequiredExternalTools() []tools.ExternalTool {
 }
 
 func (p *BicepProvider) Initialize(ctx context.Context, projectPath string, options Options) error {
-	if err := mergo.Merge(&options, Defaults); err != nil {
-		return fmt.Errorf("merging bicep defaults: %w", err)
-	}
```

Comment:
This `mergo` library is great... sad to see us removing it.  We won't be saving any time.

Reply:
I thought I was using it in more places than this.  Fine to remove it for now.  The package is very useful when working with exported types and properties which I feel is a more standard use case than with private members.